### PR TITLE
Run AppVeyor build using VS 2015 image

### PR DESCRIPTION
**Pull request by dungpa**
AppVeyor has started supporting VS 2015 on Pro environment recently http://www.appveyor.com/blog/2015/08/02/visual-studio-2015-rtm

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-os: Windows Server 2012 R2
+os: Visual Studio 2015
```
Comment (jaredpar) [inline appveyor.yml]: Why does the VS version get listed in the OS label?  Is that just how appveyor works?

Comment (dungpa) [inline appveyor.yml]: Yes, it's how AppVeyor works. See some more details at http://www.appveyor.com/updates.

Quite confusing, isn't it?

**Comment by dungpa** [comment]
Now AppVeyor build uses VS 2015 tool chain (MSBuild 14, etc.). It's a good thing going forward.

**Comment by latkin** [comment]
LGTM :+1:

**Pull request closed without merging**